javascript/src/main/components/Nav/AppNavbar.js

```diff
@@ -23,6 +23,18 @@ function AppNavbar() {
         <Navbar.Brand data-testid="brand">Mapache Search</Navbar.Brand>
       </LinkContainer>
       <Nav>
+        {isAdmin &&
+          <NavDropdown title="Admin">
+            <NavDropdown.Item href="/admin">Maintain Admins</NavDropdown.Item>
+            <NavDropdown.Item href="/admin/slackUsers">Slack Users</NavDropdown.Item>
+          </NavDropdown>
+        }
+        {(isMember || isAdmin) &&
```

Comment:
Duplicated Admin Dropdown

Reply:
You marked this as resolved, but the duplicate dropdown wasn't removed. See image:

![image](https://user-images.githubusercontent.com/34492835/101982116-94ee4880-3c26-11eb-9205-e3962c25e963.png)

Reply:
Yes.   Please do not mark things as resolved when they are not resolved.

And please don't leave code review comments unanswered.   Andrew took the time to make a code review comment.  It is fitting and appropriate to at least acknowledge that with "fixed" or "there is a reason our code is this way, and we are leaving in the duplicate: _____insert reason here____"

Just clicking "Resolved" on a code review without actually fixing the problem is ... well, it's not the way to long-term career success.  Let's just put it that way.